include/seahorn/Expr/ExprOpCore.hh

```diff
@@ -411,8 +418,13 @@ struct DefOp : public B {
   };
 
 /// macro for defining a class for a single operator
-#define NOP(NAME, TEXT, STYLE, BASE)                                           \
+#define NOP_TYPECHECK(NAME, TEXT, STYLE, BASE, TYPECHECK)                      \
```

Comment:
So just to make it clear, at the moment I'm planning to use NOP_TYPECHECK temporarily until I can change every NOP just so nothing breaks